docs/photutils/aperture.rst

```diff
@@ -160,9 +160,8 @@ them into one `~astropy.table.Table`::
 
     >>> from astropy.table import hstack
     >>> phot_table = hstack(flux)
-    >>> print(phot_table['aperture_sum_1', 'aperture_sum_2', 'aperture_sum_3'])    # doctest: +FLOAT_CMP
+    >>> print(phot_table['aperture_sum_1', 'aperture_sum_2', 'aperture_sum_3'])    # doctest: +SKIP
```

Comment:
I could not get these test for both Python 2 and 3.  The results are the "same", but strictly the table output differs because Py3 prints out more digits.  That shouldn't matter because of the `+FLOAT_CMP` comparison, but the differences in numerical output also cause the table headers to be shifted.  I *think* that's why I can't get these doctest to pass for both py2 and 3.  I went back and forth many times before giving up.

I'm now going to try explicitly setting the output format for the columns.